packages/run-protocol/test/amm/vpool-xyk-amm/test-liquidity.js

```diff
@@ -315,3 +322,63 @@ test('amm add and remove liquidity', async t => {
   poolLiquidity.secondaryAmount.value -= 8000n;
   await tracker.assertChange(poolLiquidity);
 });
+
+test('MinInitialPoolLiquidity to reserve', async t => {
+  const centralLiquidityValue = 1_500_000_000n;
+  const secondaryLiquidityValue = 300_000_000n;
+
+  // Set up central token
+  const centralR = makeIssuerKit('central');
+  const moolaR = makeIssuerKit('moola');
+  const moola = value => AmountMath.make(moolaR.brand, value);
+  const central = value => AmountMath.make(centralR.brand, value);
+
+  const electorateTerms = { committeeName: 'EnBancPanel', committeeSize: 3 };
+  // This timer is only used to build quotes. Let's make it non-zero
+  const timer = buildManualTimer(console.log, 30n);
```

Comment:
When the timer won't be used again in the current scope, I prefer to initialize it inline in the call (line 344) so as not to add its name to the local scope. In this case, there's a default value, so it's just as good to let it default, so it's never visible here. `electorateTerms` can also be moved into the call.